gf-webapp-admin/src/component/rooms/room/promo/Promo.module.scss

```diff
@@ -55,6 +55,8 @@
   }
 
   button {
```

Comment:
you already have button in library. Why do u create another style here?

Reply:
Done